### ISSUE TITLE
Rmd vs md files

### ISSUE THREAD

**Issue by adamhsparks** [adamhsparks/Reproducible-Research-in-Plant-Pathology#4]
@emdelponte, I've set up the Rmd files to knit GitHub md files, there is no need to delete them.

Rather, we should knit README.Rmd to create the README.md file in the /src directory. That will create the final output, README.md, showing the lists and other results of our methodology. This way when you go into the /src directory it will automatically display that file.

**Comment by zachary-foster** [comment]
opps, wrong button, did not mean to close.

**Comment by emdelponte** [comment]
@adamhsparks, sorry for the mess and thanks for the explanation. Actually I had cloned this repo and was committing but nothing was happening. Then, I deleted everything in my computer and forked again the right repo (from your account where I collaborate). Since I had set to knit Rmd to create the md files, I thought that I had created those files (assuming they were not there) and that is why I deleted them. Lot to learn. It is being new experience and fun to work within GitHub!
